### PR TITLE
Repository Audit by Maintainer.io

### PR DESCRIPTION
**Pull request by RichardLitt**
I've updated your README, heavily extended the Contribute, and updated the year in the license file. See email for more. Thank you! 

Feel free to comment here, or request changes.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Andrew Nesbitt
+Copyright (c) 2016-2017 Andrew Nesbitt
```
Comment (andrew) [inline LICENSE]: This can just be the current year, the project was started back in 2011

File: README.md [hunk 2]
```diff
@@ -841,23 +850,21 @@ Run the tests like this:
...
  * Send a pull request. Bonus points for topic branches.
 
+### Code of Conduct
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+
 ## Copyright
 
-Copyright (c) 2016 Andrew Nesbitt. See [LICENSE](https://github.com/splitrb/split/blob/master/LICENSE) for details.
+[MIT License](LICENSE) © 2016-2017 [Andrew Nesbitt](https://github.com/andrew).
```
Comment (andrew) [inline README.md]: Same here, no need for the year range

**Comment by andrew** [comment]
FYI this review was paid for with funds from our open collective: https://opencollective.com/split/expenses#exp1014

**Comment by RichardLitt** [comment]
Pushed changes

**Comment by andrew** [comment]
🎉  Thanks again @RichardLitt, hoping to make time to review your gist and email you back over the easter weekend 🍫

**Comment by RichardLitt** [comment]
@andrew Thank you! Looking forward to your response. And no rush. Enjoy your Friday.

**Pull request merged**